docs/core/config/mailbox/mail_location.md

```diff
@@ -27,30 +27,24 @@ See [[variable]] for a full list, but the most commonly used ones are:
 
 | Variable | Description |
 | -------- | ----------- |
-| `%u` | Full username. |
-| `%n` | User part in `user@domain`; same as `%u` if there's no domain. |
-| `%d` | Domain part in `user@domain`; empty if there's no domain. |
+| `%{user}` | Full username. |
+| `%{user | username}` | User part in `user@domain`; same as `%{user}` if there's no domain. |
+| `%{user | domain}` | Domain part in `user@domain`; empty if there's no domain. |
 
 ### Directory Hashing
 
-You can use three different kinds of hashes in [[variable]].
+Examples on how to do it:
 
-* `%N` is MD5-based "new hash" which works similarly to `%H` except it
-  gives more uniform results.
+* How to replace '%N' in new format:
```

Comment:
This is only talking about converting old format. Maybe it belongs to upgrading page? This section here should be talking about how to recommend doing the directory hashing in general. Maybe it could also mention the old ways, but it shouldn't simply start saying "How to replace old way" when people reading it wouldn't have used the old way before.